api/handlers.go

```diff
@@ -148,6 +150,18 @@ func (si *ServerImplementation) MakeHealthCheck(ctx echo.Context) error {
 // LookupAccountByID queries indexer for a given account.
 // (GET /v2/accounts/{account-id})
 func (si *ServerImplementation) LookupAccountByID(ctx echo.Context, accountID string, params generated.LookupAccountByIDParams) error {
+	rc, dpName := Verify(si.disabledParams, "LookupAccountByID", ctx, si.log)
+	switch rc {
+	case verifyIsGood:
+		// Do nothing
+		break
+	case verifyFailedParameter:
+		return badRequest(ctx, fmt.Sprintf("provided disabled parameter: %s", dpName))
+
+	case verifyFailedEndpoint:
+		return badRequest(ctx, "endpoint is disabled")
+	}
```

Comment:
Is there a way to compare the swagger "path" with the context path?

If yes, I think there is a way to implement this as a middleware. That would let us skip this boilerplate and handle it globally.

Reply:
I couldn't find a way that didn't involve codegen changes but we could talk more on a call to see if I missed something.